apps/snitch_core/lib/core/data/model/product.ex

```diff
@@ -249,4 +249,26 @@ defmodule Snitch.Data.Model.Product do
       Map.put(acc, v_id, sp)
     end)
   end
+
+  # Ordering of product depends on many things, for now we just check
```

Comment:
Make this a todo instead of a comment. Also, it would be good if you can add a small doc for this.